app/modules/ads/module.js

```diff
@@ -51,25 +51,25 @@ class Ads {
   }
 
   callExternals() {
```

Comment:
Just wonder if we shoulnd't move all externals to `PAGE_RENDER_EVENT` (for example on next pv there might be different set of slots for bidders). Anyway it sounds as quite big change so let's leave it for now